contracts/WitnetProxy.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.6;
+
+import "./utils/Upgradable.sol";
+
+/** @title WitnetProxy: upgradable delegate-proxy contract that routes Witnet data requests coming from a 
+ * `UsingWitnet`-inheriting contract to a currently active `WitnetRequestBoard` implementation. 
+ *
+ * https://github.com/witnet/witnet-ethereum-bridge/tree/0.3.x
+ *
+ * Written in 2021 by the Witnet Foundation.
+ **/
+contract WitnetProxy {
+  Upgradable public delegate;
+
+  /// @dev Constructor with no params as to ease eventual support of Singleton pattern (i.e. ERC-2470)
+  constructor () {}
+
+  /// @dev WitnetProxies will never accept direct transfer of ETHs.
+  receive() external payable {
+    revert("WitnetProxy: no ETH accepted");
+  }
+
+  /// @dev Payable fallback accepts delegating calls to payable functions.  
+  fallback() external payable { /* solhint-disable no-complex-fallback */
+    address _delegate = address(delegate);
+
+    assembly { /* solhint-disable avoid-low-level-calls */
```

Comment:
```suggestion
    assembly {
```

This lint is not needed here. Removing it results in some warnings on other lines, those can be disabled using `solhint-disable-next-line`